packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -100,6 +103,10 @@ export class DocumentService
 
 	private documentStorageService: DocumentStorageService | undefined;
 
+	public get policies(): IDocumentServicePolicies | undefined {
+		return this._policies ?? {};
```

Comment:
not clear why you do ?? {} if return type allows undefined

Reply:
Fixed